test_ppod.py

```diff
@@ -99,3 +101,29 @@ def test_filter_files_in_bucket_without_matching_file(mocked_s3):
     with pytest.raises(KeyError):
         files = filter_files_in_bucket("ppod", "download/")
         next(files)
+
+
+def test_post_files_to_pod_success(mocked_pod):
+    with open("fixtures/pod_with_namespaces.xml", "rb") as pod_with_namespaces, open(
```

Comment:
Totally optional, but since you use both of these files in multiple places between the tests and conftest, you could make them fixtures.